utils.c

```diff
@@ -3,7 +3,7 @@
 #include <string.h>
 
 int I_root;
-int logfd;
+int logfd = 1;
```

Comment:
This is a file descriptor, initializing it without `open` seems dangerous to me. But if you insist, use 2, as this `fd` is inherited by `omxplayer`, so that its `stderr` also goes to the logfile.